### PR TITLE
Fix typos, correct grammar, make language consistent

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
Changes include:
- “License” &rarr; “Licence”, as a known for consistency with BrE in the rest of the README
- Stick to second person for "Learning objectives", since you've used it at the start of the README
- Add link to making presentations with `.Rmd` files
- Clean R code a bit
- Replace “...” with “…”

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,18 +42,18 @@ of fictional attendees][personas] we created to help design the workshop.
 
 # Learning objectives
 
-At the end of the workshop I will ...
+At the end of the workshop you will ...
 
-* ... be able to create a new project that contains an analysis script
-  which takes raw data and produces figures and tables from it without
+* … be able to create a new project that contains an analysis script
```
Comment (betatim) [inline README.md:47]: what is this unicode magic? Seriously, how badly will this break if someone is brave enough to look at this in Notepad(or whatever it is people use) on Windows?

**Review by betatim — COMMENTED** [review]
Muchos muchos gracias!

**Comment by betatim** [comment]
Merging now, still interested in knowing about using ... as special character.

**Comment by RaoOfPhysics** [comment]
> still interested in knowing about using ... as special character

Just personal preference, since it is a standard character. On Mac keyboards, hit `option`+`;`.

File: README.md (hunk 1 above)
Comment (RaoOfPhysics) [reply README.md:47]: It shouldn't break, in principle!

**Pull request merged**